resources/html/404.html

```diff
@@ -15,7 +15,6 @@ <h2>Ups! Requested file not found.</h2>
 	<p>Why not try again from the top-level of
 	<ul>
 		  <li><a href="/index.html">Web User Interface</a></li>
-		  <li><a href="/docs/index.html">Documentation</a></li>
```

Comment:
Since sphinx docs have been removed this reference is not working anymore, and therefore removed here.

But it could be usefull to have the docs locally available, as there are anyway in the checkout. Does someone of ou has experience with html and knows a way to render the markdown files?
Only adding the path to the docs will just show the files and download them on selecting.

Reply:
I am not sure if we should do this. We have them all available in the web, it'll be a lot of hassle for a small use case. Why not referring to the online documentation?

Reply:
So should be sufficient like this, or do you want to add a link?

Reply:
Let's add the documentation with a reference to Github